runbotics/runbotics-orchestrator-ui/next.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    async rewrites() {
+        console.log(process.env.NEXT_PUBLIC_RUNBOTICS_ENTRYPOINT_URL);
```

Comment:
imo we should delete this console.log

is it needed to include NEXT prefix?

Reply:
we need NEXT prefix